### DISCUSSION TITLE
Where i can find the one i put in BulkWord Editor?

### DISCUSSION THREAD

**Discussion by Shinichizxc** [FrostCo/AdvancedProfanityFilter#477] [Q&A] [answered]
Is there database?

**Comment by richardfrost** [comment, marked as answer]
Are you talking about a word you added in the Bulk Word Editor? All of the filter's config is stored using the WebExtension Storage API, so if you want to retrieve it directly you'll need to see how the browser handles it. I can't think of many reasons why you would need to access it directly like that though, and you should probably use the method described below:

If you just want to see all the config that the extension uses you can open the Options page > Config tab and press the "Export" button. That will download the config as a JSON file. If you'd rather just see the JSON (instead of downloading it) click the "Inline editor" checkbox and it will be presented on that page.
